src/Util/Frame.php

```diff
@@ -49,7 +60,7 @@ public function getFile()
     }
 
     /**
-     * @return int|null
+     * @return integer|null
```

Comment:
The php 7 scalar typehints will not allow "integer". So we'd only have to change this in the future - I look atm like that's getting passed.